tests/lib/mem_alloc/testcase.yaml

```diff
@@ -2,16 +2,21 @@ tests:
   libraries.libc.minimal.mem_alloc:
     extra_args: CONF_FILE=prj.conf
     arch_exclude: posix
-    platform_exclude: twr_ke18f
+    platform_exclude: twr_ke18f native_posix_64 nrf52_bsim
     tags: clib minimal_libc userspace
   libraries.libc.newlib:
     min_ram: 16
     extra_args: CONF_FILE=prj_newlib.conf
     arch_exclude: posix
-    platform_exclude: twr_ke18f
+    platform_exclude: twr_ke18f native_posix_64 nrf52_bsim
     filter: TOOLCHAIN_HAS_NEWLIB == 1
     tags: clib newlib userspace
   libraries.libc.newlibnano:
     extra_args: CONF_FILE=prj_newlibnano.conf
     filter: CONFIG_HAS_NEWLIB_LIBC_NANO
     tags: clib newlib userspace
+  libraries.libc.minimal.mem_alloc_new:
```

Comment:
It would better to change the name to "libraries.libc.minimal.mem_alloc_nagative_testing".